resources/prometheus/prometheus-rules.yaml

```diff
@@ -291,6 +291,40 @@ spec:
             description: |
               Remaining DB instances: {{ with query "acs_fleetshard_central_db_instances_max - acs_fleetshard_central_db_instances_used" }}{{ . | first | value | humanize }}{{ end }}. A quota increase must be requested from AWS.
             sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-029-increase-aws-rds-limits.md"
+
+    - name: aws-ses
+      rules:
+        - alert: AWSSESReputationBounceRateTooHigh
+          expr: avg_over_time(aws_ses_reputation_bounce_rate_sum[1h]) > 0.05
+          for: 1s
```

Comment:
```suggestion
```
Seems redundant when using `avg_over_time(...[1h])`. It'd either choose the average over time without `for`, or the plain metric `aws_ses_reputation_bounce_rate_sum` with a bigger `for` time.